### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by ostcar** [OpenSlides/openslides-client#856]
### Discussed in https://github.com/OpenSlides/OpenSlides/discussions/6364

<div type='discussions-op-text'>

<sup>Originally posted by **111phil** February 17, 2022</sup>
We were contacted by participants with disabilities that it is quite hard for them to use OpenSlides, since it is not compatible with assistive technologies. This should be enhanced in the future. </div>

Related issues:
- #2360
- #3057

**Comment by bug0001** [comment]
Here is a resource and set of guidelines we may use :

https://developer.mozilla.org/en-US/docs/Learn/Accessibility

This section has a list of software used by people :

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility#people_with_visual_impairments

It mentions Orca (https://wiki.gnome.org/Projects/Orca) on GNU/Linux for instance.

Good technical starters are : 

[HTML: A good basis for accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)
[CSS and JavaScript accessibility best practices](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/CSS_and_JavaScript)

I consider this issue to be very important and have added the label "high".

**Comment by bug0001** [comment]
One way to begin solving this would be to ask : which areas of the application are most critical (in this context))?

Meaning : areas heavy in text, for example. The voting area is another.

**Comment by tsiegleauq** [comment]
> since it is not compatible with assistive technologies

That's a little contradictory to other information by blind people we have received before on the professional support level regarding OS3.4. Last time we checked, assisted access worked "somewhat" decently.
Anw, angular material is "usually" well compatible for screen readers and such, however, to verify that every single view works will involve a lot off effort. Especially voting might be critical.

**Comment by ostcar** [comment]
This is some sort of a meta issue. I would propose to close it and open concrete issues instead.

**Comment by bastianjoel** [comment]
With the new angular Version (#2810) there is now a new eslint plugin `plugin:@angular-eslint/template/accessibility` which could help a bit with that. 
Currently we have 89 accessibility related errors when enabling the plugin.

**Comment by ostcar** [comment]
I close this for now. If there are concrete problems, we can open issues for them.
